lib/pelion/Kconfig

```diff
@@ -89,7 +89,7 @@ config PELION_NRF_SECURITY
 config PELION_MBEDTLS
 	bool "mbedTLS built-in"
 	select OPENTHREAD_MBEDTLS if NET_L2_OPENTHREAD
-	select MBEDTLS_CFG_FILE_PROMPT_OFF
+#	select MBEDTLS_CFG_FILE_PROMPT_OFF
```

Comment:
actually, this line should be deleted completed.
The `MBEDTLS_CFG_FILE_PROMPT_OFF` was removed during rework of initial approach.

```suggestion
```

Reply:
done